packages/utils/tool-utils/src/snapshotNormalizer.ts

```diff
@@ -112,6 +122,26 @@ function getNormalizedBlobContent(blobContent: string, blobName: string): string
 		content = JSON.stringify(gcState);
 	}
 
+	/**
+	 * The legacy catch-up ops blob in merge tree DDS contains sequenced messages. These ops used to have metadata property.
+	 * However, we stopped sending the metadata property to DDS because it was a Runtime layer concept. Remove the metadata
+	 * property from the ops because latest snapshots won't have the metadata property.
+	 */
+	if (blobName === legacyCatchUpBlobName) {
```

Comment:
I'm not sure what is relationship of this block to this feature. From the comment above, it feels like it's unrelated issue. I.e. it's not caused by this PR requiring a fix in this PR.
I might be wrong, but I did not get a connection

Reply:
It is caused by this PR. I mentioned this in the PR description. Basically, now that we are sending just one message from a grouped batch, this PR removes the `metadata` property before sending it out to the data store / DDS. Legacy merge tree snapshot had a catcupOps blob that had messages with the `metadata` property.
The reference snapshot that the snapshot test compares with has the `metadata` property for these ops, the comparison now fails.